### ISSUE TITLE
login return account status, to fix confusion when creating a new account

### ISSUE THREAD

**Issue by Die4Ever** [LemmyNet/lemmy#4562]
### Requirements

- [X] Is this a feature request? For questions or discussions use https://lemmy.ml/c/lemmy_support
- [X] Did you check to see if this issue already exists?
- [X] Is this only a feature request? Do not put multiple feature requests in one issue.
- [X] Is this a backend issue? Use the [lemmy-ui](https://github.com/LemmyNet/lemmy-ui) repo for UI / frontend issues.
- [X] Do you agree to follow the rules in our [Code of Conduct](https://join-lemmy.org/docs/code_of_conduct.html)?

### Is your proposal related to a problem?

New users often get stuck trying to create their account.

### Describe the solution you'd like.

I think an account awaiting confirmation should show you the status of your account when you're logging in. Awaiting email confirmation, awaiting confirmation from admin, whatever status. This would fix extremely common issues I see with people trying to make new accounts.

### Describe alternatives you've considered.

I can't think of anything better.

### Additional context

some recent examples of confused new users getting stuck:

https://www.reddit.com/r/Lemmy/comments/1bk5c0e/cant_login_on_my_account/

https://www.reddit.com/r/Lemmy/comments/1bc4mm0/cannot_get_registered_on_lemmy/

https://www.reddit.com/r/Lemmy/comments/1b7o5lj/banned_after_signing_up/

https://www.reddit.com/r/Lemmy/comments/18l5kl4/cant_log_in_to_lemmy/

https://www.reddit.com/r/Lemmy/comments/18277ea/question_lemmy_account_creation/

https://www.reddit.com/r/Lemmy/comments/17skypv/question_cant_login_to_any_lemmy_sitescommunity/

https://www.reddit.com/r/Lemmy/comments/17p276d/i_cant_login_to_lemmy/

**Comment by Die4Ever** [comment]
would go well with "Ability for user to resent email verification" https://github.com/LemmyNet/lemmy/issues/3777

**Comment by Vince-Smith** [comment]
> The backend already returns correct error messages for these cases. It seems to be a frontend issue which should be fixed according to [LemmyNet/lemmy-ui#2341](https://github.com/LemmyNet/lemmy-ui/issues/2341).

I'm not sure this is a case of having the incorrect error message or entirely a UI problem...

I'm not seeing any mechanisms in the HTTP API (assuming [this js library](https://join-lemmy.org/api/classes/LemmyHttp.html) documents everything) that would allow me to re-request a verification.

The specific scenario I find myself in:

1. Signed up for lemmy months ago, did not verify my email at the time for some reason.
2. Revisited and attempted to log in. Cannot because my email is not verified.
  ```
  POST https://lemm.ee/api/v3/user/login
  {"error":"email_not_verified"}
  ```
3. Attempt to reset password. Cannot because my email is not verified.
  ```
  POST https://lemm.ee/api/v3/user/password_reset
  {"error":"email_not_verified"}
  ```
4. Find the old email and attempt to verify. Cannot because the token is no longer found (or valid?)
  ```
  POST https://lemm.ee/api/v3/user/verify_email
  {"error":"token_not_found"}
  ```

I was looking for an api endpoint I could call myself in lieu of it being a UI issue that would trigger a new email verification to be sent. As far as I can tell **that does not exist**, which leaves my account effectively dead in the water.

**Comment by Nutomic** [comment]
@Vince-Smith Ive opened https://github.com/LemmyNet/lemmy/issues/5191
